### ISSUE TITLE
Bug in the Move to blocks feature

### ISSUE THREAD

**Issue by jennydupuy** [WordPress/gutenberg#29190]
## Description
With Chrome, when I use the Move To feature in the block editor and I reach the beginning or the end of the article, the focus shifts to other elements of the interface, and I can no longer move the block or abandon the feature.
The bug does not appear with Firefox (85.0.2 - 64 bits).

## Step-by-step reproduction instructions
1. In Chrome, create multiple blocks.
2. Select one of the blocks.
3. Enable the Move To feature from the toolbar of this block.
4. When the blue line appears, use the keyboard arrows to reach the end or the beginning of the post.
5. Once you have reached the beginning or end, press the up or down arrow on the keyboard again.
6. The focus is lost, and when you press the up or down arrow on the keyboard, you will see the focus move to other elements of the interface. You can no longer return to the content of the post, the block can no longer be moved, and you have to double-click in a blank space in the interface to exit the feature.

## Expected behaviour
Focus should not be lost when reaching the beginning or end of the article.

## Actual behaviour
See 6. above

## Screenshots or screen recording (optional)
![move-to](https://user-images.githubusercontent.com/75045144/108640972-05260f80-749d-11eb-9028-1ecc0c0f7eef.gif)

## WordPress information
- WordPress version: 5.6.1
- Gutenberg version: 9.8.4
- Are all plugins except Gutenberg deactivated? Yes
- Are you using a default theme (e.g. Twenty Twenty-One)?  Yes

## Device information
- Device: Desktop
- Operating system: Windows 10
- Browser: Chrome Version 88.0.4324.182 (Build officiel) (64 bits)

**Comment by talldan** [comment]
I could only reproduce this using the tab key. With arrow keys input seems to be constrained correctly.

**Comment by jennydupuy** [comment]
I tested with version 5.6.2 of WordPress and the problem persists.
I also have the issue using the TAB key.

**Comment by paaljoachim** [comment]
Testing with WordPress 5.7.
Gutenberg plugin 10.2.1
Twenty Twenty One.

Added a few different blocks.
Somewhere in the middle of the page I clicked the 3 dots and selected the Move to control. The blue line showed up and I could easily move the blue line up to the beginning of the page and down to the bottom of the page without losing focus using the up and down arrow on my keyboard.
I could also click outside the page and click back and regain the blue line and continue moving it up and down the page. Clicking enter moved the block I had selected to have moved.

The feature is working very well in my testing environment.

Using tab then I could tab the blue line to the end of the page, and there I would lose the focus. But I would not call that a bug.

I think we should close this issue.
